arm-keyvault/2015-06-01/swagger/keyvault.json

```diff
@@ -0,0 +1,463 @@
+          {
+            "name": "$top",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "format": "int32",
+            "description": "Maximum number of results to return."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "enum": [ "2015-11-01" ],
+            "description": "Azure Resource Manager Api Version."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/VaultListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "SKU family name"
+        },
+        "name": {
+          "type": "string",
+          "description": "SKU name"
+        }
+      },
+      "required": [
+        "family",
+        "name"
+      ]
+    },
+    "AccessPolicyEntry": {
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Tenant ID of the principal"
+        },
+        "objectId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Object ID of the principal"
+        },
+        "applicationId": {
+          "type": "string",
+          "format": "uuid",
+          "description": " Application ID of the client making request on behalf of a principal"
+        },
+        "permissions": {
+          "$ref": "#/definitions/Permissions",
+          "description": "Permissions that the principal has for this vault"
+        }
+      }
+    },
+    "Permissions": {
+      "properties": {
+        "keys": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Permissions to keys"
+        },
+        "secrets": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Permissions to secrets"
+        }
+      }
+    },
+    "VaultProperties": {
+      "properties": {
+        "vaultUri": {
+          "type": "string",
+          "description": "URL of the vault"
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Tenant ID"
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "SKU details"
+        },
+        "accessPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccessPolicyEntry"
+          },
+          "description": "Access policies for one or more principals"
```

Comment:
> An array of 0 to 16 identities that have access to the key vault. All identities in the array must use the same tenant ID as the key vault’s tenant ID.

The MSDN doc says that the user can provide upto 16 items in the accessPolicies array.
You may want to add "maxItems": 16 as a constraint.
Take a look at the possible constraints as per the swagger schema over [here](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#patterned-fields-2). Once the page is loaded, scroll up a little and you should see the constraints.